interpreter.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import with_statement
+
+import sys
+
+from rpython.rlib import jit
+from llvm_wrapper import *
+from state import State
+
+def target(*args):
+    return main, None
+
+def jitpolicy(driver):
+    from rpython.jit.codewriter.policy import JitPolicy
+    return JitPolicy()
+
+jit_driver = jit.JitDriver (
+    greens = [],
+    reds = [])
+
+class Interpreter(object):
+
+    def __init__(self, global_state = None):
+        self.global_state = global_state
+
+    # only works for integers
```

Comment:
This part is OK for now, but my gut feeling says you will need to re-engineer this part later.